README.adoc

```diff
@@ -0,0 +1,38 @@
+# Jenkins Automated Release Flow
+
+As a part of the link:https://issues.jenkins-ci.org/browse/INFRA-910[INFRA-910] EPIC,
+https://jenkins.io/projects/infrastructure/[Jenkins Infrastructure Team] is working on a new Jenkins Core Release Environment
+which would be able to release Jenkins Weekly and LTS versions in an automatic way.
+It includes packaging, staging, and releasing stages.
+
+## Jenkins Core Release Environment
+
+The release environment is hosted within the https://github.com/jenkins-infra/openvpn[Jenkins VPN].
+It includes a Jenkins instance and a number of other services.
+If you are connected to VPN, the Jenkins instance can be accessed using this URL: https://release.ci.jenkins.io/ .
+
+Helm Chart for the Jenkins Core Release Environment is available https://github.com/jenkins-infra/charts/tree/master/charts/jenkins[here].
```

Comment:
Thi link https://github.com/jenkins-infra/charts/blob/master/helmfile.d/jenkins-release.yaml makes more sense as it includes the jenkins chart and the release environlment charts

Reply:
Will add it before the environment chart sentence

Reply:
```suggestion
Chart for Jenkins Core Release Environment environment is available link: https://github.com/jenkins-infra/charts/blob/master/helmfile.d/jenkins-release.yaml[here].
Jenkins master chart for it is available https://github.com/jenkins-infra/charts/tree/master/charts/jenkins[here].
```